src/utils/common.ts

```diff
@@ -0,0 +1,6 @@
+export const stripArrayTypeIfPresent = (typeString: string) => {
+  if (typeString && typeString.match(/\S\[\]$/u) !== null) {
```

Comment:
The linter seems to want you to do something like this:
```suggestion
  if (typeString?.match(/\S\[\]$/u) !== null) {
```